cg/services/fastq_concatenation_service/fastq_concatenation_service.py

```diff
@@ -1,28 +1,48 @@
 import logging
 from pathlib import Path
 
+from cg.constants.constants import ReadDirection
 from cg.services.fastq_concatenation_service.utils import (
-    concatenate_forward_reads,
-    concatenate_reverse_reads,
     remove_raw_fastqs,
+    concatenate_fastq_reads_for_direction,
 )
 
 LOG = logging.getLogger(__name__)
 
 
 class FastqConcatenationService:
+    """Fastq file concatenation service."""
+
+    @staticmethod
     def concatenate(
-        self,
+        sample_id: str,
         fastq_directory: Path,
         forward_output_path: Path,
         reverse_output_path: Path,
         remove_raw: bool = False,
     ):
-        temp_forward: Path | None = concatenate_forward_reads(fastq_directory)
-        temp_reverse: Path | None = concatenate_reverse_reads(fastq_directory)
+        """Concatenate fastq files for a given sample in a directory and write the concatenated files to the output path.
+
+        Args:
```

Comment:
I have seen this in lower case and capitalised. Maybe standardise

Reply:
I tried to fix that to all lower case. Will make sure to get them all!